jonathanmusila/Questioner-v1
# #163060415 Installed dependancies

## What does this PR do?
- Install the requirements 

## Description of the task to be completed
 - [x] Install requirements `requirements.txt`

## How can this be manually tested?
- Go to the repo and clone it to your machine
- Navigate to the folders to see `requirements.txt` file

### What are the relevant PT stories
#163060415
